Activities/Database/UiPath.Database.Activities/ExecuteNonQuery.cs

```diff
@@ -105,7 +105,14 @@ protected async override Task<Action<AsyncCodeActivityContext>> ExecuteInternalA
                     var currentParam = Parameters[param.Key];
                     if (currentParam.Direction == ArgumentDirection.Out || currentParam.Direction == ArgumentDirection.InOut)
                     {
-                        currentParam.Set(asyncCodeActivityContext, param.Value.Value);
+                        if (param.Value.Value != DBNull.Value)
+                        {
+                            currentParam.Set(asyncCodeActivityContext, param.Value.Value);
+                        }
+                        else
+                        {
+                            currentParam.Set(asyncCodeActivityContext, null);
```

Comment:
I would remove the condition on else
if there is nothing to set, I would not set null; 
also null assignment wouldn't work if the type is a value type (eg. integer or bool);
eg.: 
object o = null;
bool v = (bool)o;